webpack/scenes/ContentViews/__tests__/hundredContentViews.fixtures.json

```diff
@@ -0,0 +1,1514 @@
+{
```

Comment:
The sheer size of this seems wasteful.  Would it work to create this data programmatically?

Reply:
update: don't mind me, I just read your update above.. seems like we're on the same page :yum: